### PR TITLE
#129 //solved

### PR DESCRIPTION
**Pull request by JAgrit20**

### CODE REVIEW

File: website/static/css/docusaurus-admonitions.css [hunk 1]
```diff
@@ -107,4 +107,8 @@
 .features.container.paddingBottom.paddingTop .blockImage img{
   width: 80px;
   height: 80px;
+}
+.headerTitleWithLogo{
+  display:inline-block;
+   width:100px
```
Comment (nodece) [inline website/static/css/docusaurus-admonitions.css:113]: Do you format the code? @JAgrit20 J

**Comment by nodece** [comment]
@JAgrit20 Do you want to fix the DCO? Please do not close or open repeatedly.

**Comment by JAgrit20** [comment]
> @JAgrit20 Do you want to fix the DCO? Please do not close or open repeatedly.

yeah sorry for that I formatted the code,
fixing DCO now

File: website/static/css/docusaurus-admonitions.css (hunk 1 above)
Comment (JAgrit20) [reply website/static/css/docusaurus-admonitions.css]: sir my continuous-integration/travis-ci/pr — The Travis CI build failed
what can I do now

Comment (nodece) [reply website/static/css/docusaurus-admonitions.css]: ignore.

**Pull request merged**